apps/storefront/pages/bloggen/2024/fluid-typography.mdx

```diff
@@ -0,0 +1,72 @@
+import { BlogArticleLayout } from '@layouts';
+import { Image } from '@components';
+import { Contributors } from '@blog';
+
+export default ({ children }) => (
+  <BlogArticleLayout
+    content={children}
+    heading='Dynamiske tekst-størrelser og hvorfor vi gikk bort fra det'
+    ingress='Det er mulig å bruke dynamiske tekst-størrelser uten breakpoints, men det medfører noen ulemper.  '
+    date='26. april 2024'
+    author='Designsystemet'
+    imageSrc='/img/bloggen/fluid-front.png'
+    imageAlt='Illustrasjonen viser at en 16px tekststørrelser på små skjermer øker gradvis opp til 18px på store skjermer.'
+    imageCaption='Illustrasjonen viser at en 16px tekststørrelser på små skjermer øker gradvis opp til 18px på store skjermer. '
+  />
+);
+
+Ved hjelp av “Fluid typography” kan vi skalere tekststørrelsen gradvis mellom en min og maks-bredde, i stedet for at det blir en brå oppskalering på et spesifikt brekkpunkt. Den lineære funksjonen tilnærmer seg ønsket størrelse jevnt over skjermens bredde.
+
+Tekststørrelsene øker gradvis fra viewport-bredde 320px helt til det treffer maksbredden på viewporten som er satt til 1360px. En tekst på 16px vil f.eks øke til 18px på den største viewporten.
+
+<Image
+  src='/img/bloggen/fluid-ex.png'
+  alt='Illustrasjonen viser at en 16px tekststørrelser på små skjermer øker gradvis opp til 18px på store skjermer.'
+  caption='Tekststørrelsen tilpasser seg automatisk bredden på skjermen.'
+  boxShadow={false}
+/>
+
+## Fordeler
+
+- Du slipper å tenke på breakpoints, størrelsen på skriften blir automatisk tilpasset skjermen.
+- Når størrelsene ikke justeres på konkrete breakpoints unngår vi at det blir stykkevis og brå oppskaleringer.
+
+## Ulemper
+
+- Halve piksler. Figma runder nedover. Noen nettlesere runder oppover, andre nedover. Det gjør at ting ikke ser likt ut i Figma og i nettleseren.
+- Du kan føle du har mindre kontroll.
+- I Figma kan denne størrelsesendringen kun skje ved bruk av pluginen "Token Studio", når du har aktivert ønsket viewport for din frame.
+- Det er ikke gitt at et produkt vil bygge sitt produkt ved hjelp av dynamisk typografi. Dersom noen legger til moduler fra designsystemet i eget grensesnitt vil det ikke samsvare med skaleringen deres.
+
+<br />
+
+**Basert på [ulempene dette medførte](https://github.com/digdir/designsystemet/issues/1444), valgte vi å gå for en [statisk typografi-skala](/grunnleggende/designelementer/typografi#tekststørrelse) som standard for designsystemet.**
```

Comment:
```suggestion
Basert på [ulempene dette medførte](https://github.com/digdir/designsystemet/issues/1444), valgte vi å gå for en [statisk typografi-skala](/grunnleggende/designelementer/typografi#tekststørrelse) som standard for designsystemet.
```
I think we should remove bold here, or update our css to target `strong a`, so we get the correct styling for links. It is currently different:
![image](https://github.com/digdir/designsystemet/assets/14057205/50147ae1-e8e2-42e5-8d79-ee046d7db0a4)